tests/test_query_across_tables.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+import synapseclient as sc
+from synapsebridgehelpers import query_across_tables
+
+syn = sc.login()
+
+TESTING_TABLE = "syn18503475"
+TESTING_TABLE_2 = "syn18503678"
+
+def test_str_query():
+    query_across_tables(syn, tables = [TESTING_TABLE, TESTING_TABLE_2],
+                        query = "select * from %s")
+
```

Comment:
YOu aren't verifying the returned data - why perform this as integration test?

Reply:
note to self: TODO